Services/WalletConnectorService/Sources/WalletConnectorService.swift

```diff
@@ -252,7 +246,7 @@ extension WalletConnectorService {
     private func signTransaction(chain: Chain, request: WalletConnectSign.Request) async throws -> RPCResult {
         let params = try request.params.get([Primitives.WCEthereumTransaction].self)
         guard let transaction = params.first else {
-            throw AnyError("error")
+            throw AnyError("Wrong wallet connect sign parameters")
```

Comment:
```suggestion
            throw AnyError("Wrong wallet connect sign parameters")
```

Create `WalletConnectorError: Error`, as this is a big module and has many errors